### DISCUSSION TITLE
Feature: Master Key

### DISCUSSION THREAD

**Discussion by T3chie-404** [phantom/docs#177] [General]
Hello Team...As more and more newbies coming into crypto are falling victim to wallet drainers that change their stake / withdraw authority of their staked SOL accounts...we need to somehow adapt to help protect the community or it will never be able to be mass-adopted as we all want. 

Would it be possible to add a "Master Signing Key" that becomes the stake/withdraw authority when a phantom wallet is created...therby needing a separate signing key (and separate password) to make changes to sensitive account actions such as changing the staker authority or withdraw authority? 

Then the standard transactions could continue to happen with a single signer, but if a wallet drainer is trying to change more sensitive information, they are prompted with a warning and password request to elevate the transaction authority to the master key?

**Comment by adamdelphantom** [comment]
Hello @T3chie-404,

Thank you for writing in with this!

Please consider posting your feature request [here](https://phantom.canny.io). So that others may see it and vote if they would like to see such a feature in Phantom.
